### DISCUSSION TITLE
How to change default width and height for flux template

### DISCUSSION THREAD

**Discussion by alaman64** [lllyasviel/stable-diffusion-webui-forge#1548] [Q&A]
would like to change to something other than 896 x 1152. Used to go in settings and defaults, and change it there. no that no longer works, when I restart it calls up my saved settings, I see them for about 1 second, and then they are over written by 896 x 1152. any ideas?

**Comment by DenOfEquity** [comment]
*Settings* > *UI defaults 'flux'*

**Reply by reghope** [reply]
> _Settings_ > _UI defaults 'flux'_

Thanks. Didn't see that! 

Edit: Hadn't updated my install - apologies.

**Comment by abzaloff** [comment]
> would like to change to something other than 896 x 1152. Used to go in settings and defaults, and change it there. no that no longer works, when I restart it calls up my saved settings, I see them for about 1 second, and then they are over written by 896 x 1152. any ideas?

Settings => UI defaults 'flux' => txt2img width, txt2img height => Apply setting => Reload UI
